bigquery/google/cloud/bigquery/table.py

```diff
@@ -808,18 +824,38 @@ def items(self):
         return items
 
     def get(self, key, default=None):
-        """
-        Return value under specified key
-        Defaults to None or specified default
-        if key does not exist:
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('x')
-        'a'
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z')
-        None
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z', '')
-        ''
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z', default = '')
-        ''
+        """Return value for ``key`` if ``key`` is in the row, else
+        ``default``.
```

Comment:
Please follow [PEP257][1] and keep the one-line summary to one line.

[1]: https://www.python.org/dev/peps/pep-0257/

Reply:
Done.